src/test/a11y/test-suite.js

```diff
@@ -34,19 +34,13 @@ const dateIn3Months = () => {
   Runs though the application, evaluating each page and performing post requests to populate the necessary
   data in the database.
  */
-const runTests = async () => {
+const runEndToEndTest = async (results) => {
```

Comment:
@dwybourn this means the patch I sent you won't work without some changes

Reply:
Ok. I'll apply and fix after this PR has gone in.